src/docs/markdown/caddyfile/directives/reverse_proxy.md

```diff
@@ -6,6 +6,17 @@ title: reverse_proxy (Caddyfile directive)
 
 Proxies requests to one or more backends with configurable transport, load balancing, health checking, header manipulation, and buffering options.
 
+- [Syntax](#syntax)
+- [Upstreams](#upstreams)
+- [Load balancing](#load-balancing)
+  - [Active health checks](#active-health-checks)
+  - [Passive health checks](#passive-health-checks)
+- [Streaming](#streaming)
+- [Headers](#headers)
+- [Transports](#transports)
+  - [The `http` transport](#the-http-transport)
+  - [The `fastcgi` tranport](#the-fastcgi-transport)
+- [Examples](#examples)
```

Comment:
Is it worth the investment of writing ToC extension for goldmark in Caddy? Hugo has done something similar

Reply:
:man_shrugging: I don't think these are hard to write, and if we need them we can type them up in a minute or two. Also not that hard to maintain, these aren't high-churn pages. But I mean, if you feel like writing code, I won't stop you :joy:

Reply:
I don't have much of a preference. We have some room on the right side of the page if we want to ToC every page. But yeah, it does seem like a lot of work to set that up.